docs/process/first-responder.md

```diff
@@ -5,16 +5,20 @@ We have a first responder rotation. The goals of the rotation are:
 1. Ensure community issues and PRs are answered in a timely manner.
 1. Ensure support load is shared across the whole team.
 1. Free up the rest of the team to focus on milestone work.
-1. Allocate time to less prioritized work.
+1. Give everyone a regular break from milestone work.
 
-Each rotation is a week long. While first responder your duties are:
+Each rotation is a week long. While first responder your primary duties are:
 
 1. Triage issues.
-  * Troubleshoot or follow up on troubleshooting started by the previous first responder.
-  * Review the [more-information-needed](https://github.com/desktop/desktop/issues?q=is%3Aopen+is%3Aissue+label%3Amore-information-needed+sort%3Aupdated-asc) and [needs-reproduction](https://github.com/desktop/desktop/issues?q=is%3Aopen+is%3Aissue+sort%3Aupdated-asc+label%3Aneeds-reproduction) issues and close any that have gone 2 weeks with no new activity.
-  * See https://github.com/desktop/issue-triage/blob/master/issue-triage.md for more information on our issue triage process.
+    * Troubleshoot or follow up on troubleshooting started by the previous first responder. The current first responder should always bear responsibility for pushing troubleshooting forward, unless another team member has explicitly taken ownership.
+    * Ensure issues are labeled accurately.
+    * Review the [`more-information-needed`](https://github.com/desktop/desktop/issues?q=is%3Aopen+is%3Aissue+label%3Amore-information-needed+sort%3Aupdated-asc) and [`needs-reproduction`](https://github.com/desktop/desktop/issues?q=is%3Aopen+is%3Aissue+sort%3Aupdated-asc+label%3Aneeds-reproduction) issues and close any that have gone 2 weeks with no new activity.
+    * See [issue-triage.md](issue-triage.md) for more information on our issue triage process.
+1. Check community pull requests and label ones that are `ready-for-review`.
 1. Review community pull requests.
-1. Address technical debt, pet bugs, etc.
-1. Anything that supports the goals outlined above.
 
-It's kind of a [big block of cheese](https://www.youtube.com/watch?v=Vm9HZq53rqU) situation.
```

Comment:
Terribly sad to see this reference go but I'll get over it.

Reply:
![czqoolf](https://user-images.githubusercontent.com/13760/32779251-fd26fb78-c90a-11e7-907a-f2a5372c0ee2.gif)